templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -574,7 +580,7 @@ server {
     ---- Code below is shared between HTTP and HTTPS ----
 #}
 {% if (not nginx_tpl_ssl and ((item.redirect is defined and item.redirect) or (item.redirect_to_ssl is defined and item.redirect_to_ssl))) %}
-        return {{ item.redirect_code | default(nginx_tpl_default_redirect_code) }} {{ nginx_tpl_http_redirect }};
+        return {{ item.redirect_code_ssl | default(nginx_tpl_default_redirect_code_ssl) }} {{ nginx_tpl_http_redirect }};
```

Comment:
It's in both sections. Anywhere where you use `nginx_tpl_http_redirect`, it should be `301`.